data/index.html

```diff
@@ -1,29 +1,145 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
-  <title>Solar System</title>
-  <link rel="stylesheet" href="index.css">
+ <meta charset="UTF-8">
+ <title>Solar System Explorer</title>
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
```

Comment:
Links to off-system resources unwelcome; these activities are to function without internet.